src/components/NavBar/NavBarView.js

```diff
@@ -97,7 +97,7 @@ const NavBar = ({
 
           {/* End Sidebar button */}
           <div id="navbar" className={classes.buttonContainer}>
-            {navBarData.length < 5 ? navBarData.map((navButton) => (
+            {navBarData.slice(0, 4).map((navButton) => (
```

Comment:
Can limit it as a max variable in custodian file to align like in case detail and other pages